src/cumulus/cli.py

```diff
@@ -60,14 +62,16 @@ def main():
                        for command in commands if command[0] != 'Base'][0]
             command = command(options)
             command.run()
+            clean_logs()
             return
-
     exit(__doc__)
 
 
 def start_container(service):
     if service:
+        print ('Starting {0}...'.format(service))
         subprocess.call([DOCKER_COMPOSE, "up", "-d", service])
+        print ('{0} Started!'.format(service))
```

Comment:
I think this is outdated, I provide the url they can hit to see the web app.

Reply:
It is outdated. I have to merge manually, so that's probably one of the reasons why.